plextraktsync/sync.py

```diff
@@ -68,12 +68,34 @@ def sync(self, walker: Walker, dry_run=False):
                 self.sync_watched(movie, dry_run=dry_run)
                 if not is_partial:
                     listutil.addPlexItemToLists(movie)
-                    if self.config.clear_collected:
-                        movie_trakt_ids.add(movie.trakt_id)
+                    movie_trakt_ids.add(movie.trakt_id)
 
-            if movie_trakt_ids:
+            if self.config.clear_collected and movie_trakt_ids:
                 self.clear_collected(self.trakt.movie_collection, movie_trakt_ids)
 
+            remaining_movies_ids = (
+                set(self.trakt.watched_movies.keys()).union(
+                    set(self.trakt.ratings["movies"])
+                ) - movie_trakt_ids
+            )
+
+            if remaining_movies_ids and not is_partial and self.config["plex_online"]:
+                """Sync ratings and watched status of movies not in Plex library"""
+                items = set(self.trakt.watched_movies.values()).union(
+                    set(self.trakt.ratings.items["movies"])
+                )
+                # items is a set() of trakt.movies.Movies already watched or rated (can a user rate without watch?)
```

Comment:
you can definitely rate movie without watching it:
- https://trakt.tv/movies/society-of-the-snow-2023